app/src/main/java/org/jellyfin/mobile/bridge/ExternalPlayer.kt

```diff
@@ -334,7 +334,7 @@ class ExternalPlayer(
                 ComponentName(packageName, "$packageName.ActivityScreen")
             }
             ExternalPlayerPackage.VLC_PLAYER -> {
-                ComponentName(packageName, "$packageName.gui.video.VideoPlayerActivity")
+                ComponentName(packageName, "$packageName.StartActivity")
```

Comment:
Do we want to add a version check here so users with an older version of the VLC app don't need to update on Android 12<=?

Reply:
I don't think so, it'll still work, just not have progress tracking.
Also I'm not even sure the recent VLC updates didn't break it anyway on all versions.